packages/ui-extensions/src/surfaces/checkout/targets.ts

```diff
@@ -709,6 +709,13 @@ export interface RenderExtensionTargets {
     CheckoutApi & StandardApi<'purchase.checkout.footer.render-after'>,
     AnyComponent
   >;
+  /**
+   * A static extension target that is rendered on top of the Checkout page as an overlay.
```

Comment:
Should we update the copy considering that the target will be restricted to the bottom right corner?